README.md

```diff
@@ -593,6 +594,7 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
   * [Docker Compose](https://docs.docker.com/compose/) - Define and run multi-container Docker applications.
   * [Docker Swarm](https://docs.docker.com/engine/swarm/) - Manage cluster of Docker Engines.
   * [Portainer](https://portainer.io/) - Simple management UI for Docker.
+  * [Yacht](https://yacht.sh/) - A container management UI with a focus on templates and 1-click deployments.
```

Comment:
Still in alpha; see https://github.com/awesome-foss/awesome-sysadmin/pull/357 and https://github.com/awesome-foss/awesome-sysadmin/blob/master/.github/PULL_REQUEST_TEMPLATE.md, this would not have passed the review under the previous maintainer and I don't want to change the way the list is maintained right now (it's in minimal maintenance mode, I'm just doing some cleanup). Feedback welcome in the thread I linked above.

Please remove it from your patch.

Reply:
please remove yacht as per the comment above